src/mlpack/core/arma_extend/Mat_extra_bones.hpp

```diff
@@ -29,136 +29,4 @@ typedef Row<elem_type>   row_type;
  * row_col_iterator for Mat<eT>. This iterator can return row and column index
  * of the entry its pointing too. The functionality of this iterator is similar
  * to sparse matrix iterators.
```

Comment:
We can remove the comments too since we no longer need to add row_col_iterator to Mat. :)